config/application.rb

```diff
@@ -48,8 +52,7 @@ class Application < Rails::Application
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
     config.time_zone = "UTC"
 
-    # The default locale is :en and all translations from config/locales/**/*.rb,yml are auto loaded.
-    config.i18n.load_path += Dir[Rails.root.join("config/locales/**/*.{rb,yml}")]
+    # The default locale is :en.
```

Comment:
The load_path configuration defaults to this since Rails 7.0: https://guides.rubyonrails.org/v7.1/configuring.html#config-i18n-load-path